private/locale-pack/package.json

```
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "name": "locale-pack",
+  "author": "Merlijn Vos <merlijn@transloadit.com>",
+  "description": "Generate locale pack, types, and documentation",
+  "main": "index.mjs",
+  "scripts": {
+    "build": "yarn node index.mjs",
+    "test": "yarn node test.mjs"
+  },
+  "dependencies": {
```

We should note them as dev dependencies, so they are correctly marked as such in the lock file (not sure if that makes an actual difference in practice tbh, but doesn't cost much to do)
```suggestion
  "devDependencies": {
```

---

eslint will give me warnings they would need to be moved to normal dependencies. But conceptually I also think they belong in `dependencies`, as this is now it's own module, therefor all the deps used are 'prod' and not dev.